### PR TITLE
[15.0][FIX] module_analysis: Fix tests

### PR DESCRIPTION
**Pull request by victoralmau**
Prevent import error

`ImportError: cannot import name 'at_install' from 'odoo.tests.common' (/opt/odoo/odoo/tests/common.py)`

Please @pedrobaeza can you review it?

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This is still red

File: module_analysis/tests/test_module.py [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests.common import TransactionCase, at_install, post_install
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
 
 
-@at_install(False)
-@post_install(True)
+@tagged("post_install")
```
Comment (pedrobaeza) [inline module_analysis/tests/test_module.py]: AFAIK, this should be:

```suggestion
@tagged("-at_install", "post_install")
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
CI is still red. And AFAIK, you can import both `TransactionCase` and `tagged` from the same package.

**Comment by pedrobaeza** [comment]
Still red.

**Comment by pedrobaeza** [comment]
cherry-pick #2625

**Comment by victoralmau** [comment]
Changes done.

**Comment by pedrobaeza** [comment]
Great, let's fix the tests:

/ocabot merge patch

**Pull request merged**